cdap-ui/app/features/home/routes.js

```diff
@@ -51,7 +51,7 @@ angular.module(PKG.name+'.feature.home')
           if(!n.length) {
             mySessionStorage.get(PREFKEY)
               .then(function (latest) {
-                var d = latest || rNsList[0].name;
+                var d = latest || 'default';
```

Comment:
isn't there a variable somewhere? Also, I'm not sure about UI code, but we should make sure that the default namespace exists, if not, bail out with a dialog saying 'default namespace not found. Please try restarting CDAP.' or something to that effect.

Thoughts?